randobot/handler.py

```diff
@@ -180,14 +178,24 @@ async def ex_s7(self, args, message):
 
         Set up room for Draft Mode.
         """
-        if self._race_in_progress() or not self._is_s7_race():
+        if self._race_in_progress():
             return
         
         draft = self.state.get('draft_data')
 
         # Handle valid arguments.
-        if len(args) >= 1 and args[0] in ('tournament', 'practice', 'qualifier', 'cancel'):
-            if args[0] in ('tournament', 'practice') and not draft.get('enabled'):
+        if len(args) == 0:
+            await self.send_message(
+                'Invalid format. Use !s7 <tournament|practice|cancel>.'
+            )
+            return
+        elif len(args) >= 1 and args[0] in ('tournament', 'practice', 'qualifier', 'cancel'):
+            if args[0] in ('tournament', 'qualifier') and not self._is_s7_race():
+                await self.send_message(
+                    'This is not an official tournament room. Use !practice <draft|auto> instead.'
```

Comment:
Typo - `!practice` should be `!s7 practice`

Reply:
😆 whoops